code/modules/admin/DB ban/functions.dm

```diff
@@ -566,4 +572,11 @@ datum/admins/proc/DB_ban_unban_by_id(var/id)
 
 			output += "</table></div>"
 
-	usr << browse(output,"window=lookupbans;size=900x700")
\ No newline at end of file
+	usr << browse(output,"window=lookupbans;size=900x700")
+
+/proc/flag_account_for_forum_sync(var/ckey)
+	if(!dbcon)
+		return
+	var/sql = "UPDATE [format_table_name("player")] SET fupdate = 1 WHERE ckey = '[ckey]'"

```

Comment:
A little nervous about possible privilege escalation attacks where `ckey` contains a SQL injection, since this proc on its own does not do any sanitization - `ckey` should be fine for sanitizing it

Reply:
Yeah, I was nervous too, but this proc is only called from other procs that already sanitize. 
But, I'll add a sanitizeSQL() to it anyway, why not. It can't hurt.